parser/__tests__/index.test.js

```diff
@@ -378,3 +378,128 @@ describe('trailing whitespace', () => {
     });
   });
 });
+
+describe('inline image', () => {
+  test('with alt text', () => {
+    expect('![test](https://example.com/image.png)').toBeParsedAs([
+      {type: 'syntax', start: 0, length: 1},
+      {type: 'syntax', start: 1, length: 1},
+      {type: 'syntax', start: 6, length: 1},
+      {type: 'syntax', start: 7, length: 1},
+      {type: 'link', start: 8, length: 29},
+      {type: 'syntax', start: 37, length: 1},
+    ]);
+  });
+
+  test('without alt text', () => {
+    expect('![](https://example.com/image.png)').toBeParsedAs([
+      {type: 'syntax', start: 0, length: 1},
+      {type: 'syntax', start: 1, length: 1},
+      {type: 'syntax', start: 2, length: 1},
+      {type: 'syntax', start: 3, length: 1},
+      {type: 'link', start: 4, length: 29},
+      {type: 'syntax', start: 33, length: 1},
+    ]);
+  });
```

Comment:
Let's add a newline here
```suggestion
  });

```